spec/integ/megolm-integ.spec.ts

```diff
@@ -1010,70 +1010,73 @@ describe("megolm", () => {
         expect(decryptedEvent.getClearContent()).toBeUndefined();
     });
 
-    it("should successfully decrypt bundled redaction events that don't include a room_id in their /sync data", async () => {
-        await aliceTestClient.start();
-        aliceTestClient.client.crypto!.deviceList.downloadKeys = () => Promise.resolve({});
-        const p2pSession = await createOlmSession(testOlmAccount, aliceTestClient);
-        const groupSession = new Olm.OutboundGroupSession();
-        groupSession.create();
-
-        aliceTestClient.client.crypto!.deviceList.getUserByIdentityKey = () => "@bob:xyz";
-
-        // make the room_key event
-        const roomKeyEncrypted = encryptGroupSessionKey({
-            senderKey: testSenderKey,
-            recipient: aliceTestClient,
-            p2pSession: p2pSession,
-            groupSession: groupSession,
-            room_id: ROOM_ID,
-        });
-
-        // encrypt a message with the group session
-        const messageEncrypted = encryptMegolmEvent({
-            senderKey: testSenderKey,
-            groupSession: groupSession,
-            room_id: ROOM_ID,
-        });
+    it(
+        "should successfully decrypt bundled redaction events " + "that don't include a room_id in their /sync data",
```

Comment:
Looks silly, but if you remove the concat, prettier puts `it`, the string and `async () => {` into a single line. Then ESLint starts complaining about hitting the hard line length limit. :man_shrugging:

Reply:
If prettier doesn't comprehend our hard line limit then this will be really painful to have to conjure

Reply:
They added something about it in their docs
https://prettier.io/docs/en/options.html#print-width

As far as I understand that means use prettier or disable length checks in your other linters.

Reply:
Sounds like we'll need to ask for people's opinions on this, conveniently it is thursday

Reply:
We decided to disable `max-len` in ESLint